alpaca/broker/client.py

```diff
@@ -525,7 +525,7 @@ def _get_account_activities_iterator(
             last_result = result[-1]
 
             if "id" not in last_result:
-                raise APIError(
+                raise AssertionError(
```

Comment:
built-in `AssertionError` is defined as `Raised when an assert statement fails.`.
Could be something different exception?

ref. https://docs.python.org/3/library/exceptions.html#AssertionError

Reply:
Maybe we can raise `ValueError` or `[AttributeError](https://docs.python.org/3/library/exceptions.html#AttributeError)`? Which one it's better in your opinion? I'd go with the attribute error based on the docs